configure

```diff
@@ -24,6 +24,8 @@ ipcsocket=''
 pcdir='$prefix/lib/pkgconfig'
 systemlibdirs="/lib64 /usr/lib64 /usr/local/lib64 /lib /usr/lib /usr/local/lib"
 sysroot=''
```

Comment:
I'm curious what @dequis or @Wilm0r  thing about this change; I'm personally hesitant to avoid that  all of these modern shell constructs just to please the shell on Solaris 10. Solaris 10 is ancient - that's like trying to keep support for Debian sarge around.

There's nothing to ensure Solaris 10 compatibility stays put, and given the number of lines in configure that this touches we're very likely to break support next time we make a change to configure. And even if we remember, there's no good way to test that we don't break it.

Reply:
I _think_ /bin/sh is as broken on Solaris 11 however I'll verify that tomorrow too, if that makes any difference.

This is of course possible to work around just doing #!/usr/bin/env sh (so /usr/xpg4/bin/sh can be used, or whatever is put in the path) or executing the script like bash ./configure,

Reply:
I was wrong with the shell on Solaris 11, the existing configure script works just fine. Judging from the Illumos man pages it should be fine on other Solaris derivates as well.

Want me to drop it from the PR?

Reply:
Yeah, please do - or perhaps create a separate PR for it if you do feel strongly about it. That way we can at least land these changes and perhaps have further discussion about the shell compat.

Reply:
I've removed those changes from the PR now.